Kickstarter-iOS/Assets.xcassets/icons/icon--currency-header.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "icon--currency-header.pdf"
```

Comment:
Since I'm not familiar with our naming conventions, what is the reason for using 2 dashes? `icon--curr...`

Reply:
Agreed! I'm pretty sure this started as an error and then became convention. @Scollaco tells me otherwise 😂